tests/details/lending-bar.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { test } from '../fixtures';
+
+test(`Load theater: bookreader with lending bar`, async ({ lendingBar, loginPage }) => {
+  await test.step('loan book with basic components', async () => {
+    await lendingBar.gotoPage('coleridgesublime0000char');
+    await lendingBar.verifyDefaultTexts();
```

Comment:
if we use `detailsPage` fixture, this can be something like: `detailsPage.lendingBar.verifyDefaultTexts()`